src/bitcoin-cli.cpp

```diff
@@ -278,20 +273,20 @@ class AddrinfoRequestHandler : public BaseRequestHandler
         if (!nodes.empty() && nodes.at(0)["network"].isNull()) {
             throw std::runtime_error("-addrinfo requires bitcoind server to be running v22.0 and up");
         }
-        // Count the number of peers known to our node, by network.
-        std::array<uint64_t, NETWORKS.size()> counts{{}};
         for (const UniValue& node : nodes) {
-            std::string network_name{node["network"].get_str()};
-            const int8_t network_id{NetworkStringToId(network_name)};
-            if (network_id == UNKNOWN_NETWORK) continue;
-            ++counts.at(network_id);
+            std::string network{node["network"].get_str()};
+            if (m_counts.find(network) == m_counts.end()) {
+                m_counts.insert({network, 0});
+            }
+            ++m_counts.find(network)->second;
```

Comment:
```suggestion
            ++m_counts[network];
```

Reply:
thanks, fixed in bf69a13677680dbf2065e18e3e5c7882ec3aab33